code/script.js

```diff
@@ -1,9 +1,68 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
+// function toggle() {
+//   this.classList.toggle("active")
+// }
+
+// // Selects an HTML element, and calls a function which will be executed when the element is clicked.
+// document.getElementById("section1").onclick = toggle
+
+
+const accordionItemHeader1 = document.getElementById('accordionItemHeader1');
+const accordionItemHeader2 = document.getElementById('accordionItemHeader2');
+const accordionItemHeader3 = document.getElementById('accordionItemHeader3');
+
+accordionItemHeader1.onclick = () => {
+  const accordionTextContainer1 = accordionItemHeader1.nextElementSibling;
+  accordionTextContainer1.classList.toggle('active');
+  accordionItemHeader1.classList.toggle('active');
+
+  const accordionTextContainer2 = accordionItemHeader2.nextElementSibling;
+  accordionTextContainer2.classList.remove('active');
+  accordionItemHeader2.classList.remove('active');
+  
+  const accordionTextContainer3 = accordionItemHeader3.nextElementSibling;
+  accordionTextContainer3.classList.remove('active');
+  accordionItemHeader3.classList.remove('active');
 }
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+accordionItemHeader2.onclick = () => {
+  const accordionTextContainer2 = accordionItemHeader2.nextElementSibling;
+  accordionTextContainer2.classList.toggle('active');
+  accordionItemHeader2.classList.toggle('active');
+
+  const accordionTextContainer1 = accordionItemHeader1.nextElementSibling;
+  accordionTextContainer1.classList.remove('active');
+  accordionItemHeader1.classList.remove('active');
+
+  const accordionTextContainer3 = accordionItemHeader3.nextElementSibling;
+  accordionTextContainer3.classList.remove('active');
+  accordionItemHeader3.classList.remove('active');
+}
+
+accordionItemHeader3.onclick = () => {
+  const accordionTextContainer3 = accordionItemHeader3.nextElementSibling;
+  accordionTextContainer3.classList.toggle('active');
+  accordionItemHeader3.classList.toggle('active');
+  
+
+  const accordionTextContainer1 = accordionItemHeader1.nextElementSibling;
+  accordionTextContainer1.classList.remove('active');
+  accordionItemHeader1.classList.remove('active');
+  
+  const accordionTextContainer2 = accordionItemHeader2.nextElementSibling;
+  accordionTextContainer2.classList.remove('active');
+  accordionItemHeader2.classList.remove('active');
+}
```

Comment:
Since this code looks almost the same for all `accordionItemHeader`s , it would be cool if we can shorten it without repeating ourselves. If you want, take a look at Maksymilian's notes from Week 3 lecture 2 (where he does a `forEach` loop) :)